### PR TITLE
Adds bot.replyIntent()

### PR DESCRIPTION
**Pull request by RafaelCosman**
Usage:

`bot.replyIntent(message, "greeting");`

Will hit `http://www.chatamize.com/api/v1/intent?api_key=[your api key]&intent=greeting` and will reply with the message returned. See `chatamize.com` to manage your messages.

As of `73374c0`, `intents-chatamize` passes all tests with `npm test`.
```
Test
    ✓ should have a token
    ✓ should have Botkit instance

  Botkit
info: ** No persistent storage method specified! Data may be lost when process shuts down.
info: ** Setting up custom handlers for processing Slack messages
info: ** API CALL: https://slack.com/api/rtm.start
notice: ** BOT ID:  intent-wit-bot  ...attempting to connect to RTM!
    ✓ should start and then stop (1782ms)
info: ** No persistent storage method specified! Data may be lost when process shuts down.
info: ** Setting up custom handlers for processing Slack messages
info: ** API CALL: https://slack.com/api/rtm.start
    ✓ should have fail with false token (171ms)

  Log
    ✓ should use an external logging provider (177ms)


  5 passing (2s)
```

### CODE REVIEW

File: lib/Slackbot_worker.js [hunk 1]
```diff
@@ -358,6 +358,45 @@ module.exports = function(botkit, config) {
...
+            });
+
+            //the whole response has been recieved, so we just print it out here
+            response.on('end', function() {
+                response = JSON.parse(str);
+                bot.reply(src, response.message, cb);
+            });
+        };
+
+        console.log('sending request');
```
Comment (anonrig) [inline lib/Slackbot_worker.js]: Unnecessary `console.log`

File: lib/Slackbot_worker.js [hunk 2]
```diff
@@ -358,6 +358,45 @@ module.exports = function(botkit, config) {
...
+
+        // see https://docs.nodejitsu.com/articles/HTTP/clients/how-to-create-a-HTTP-request
+        var http = require('http');
+
+        var options = {
+            host: 'www.chatamize.com',
+            path: '/api/v1/message?' + dataString
+        };
+
+        console.log(options.path);
```
Comment (anonrig) [inline lib/Slackbot_worker.js]: Unnecessary `console.log`

File: lib/Slackbot_worker.js [hunk 3]
```diff
@@ -358,6 +358,45 @@ module.exports = function(botkit, config) {
...
+        if (!bot.config.token_chatamize) {
+            console.log('Please set token_chatamize in controller.spawn().');
+            return;
+        }
+
+        dataString = '';
+        dataString += 'api_key=' + bot.config.token_chatamize;
+        dataString += '&intent=' + intent;
+
+        // see https://docs.nodejitsu.com/articles/HTTP/clients/how-to-create-a-HTTP-request
```
Comment (anonrig) [inline lib/Slackbot_worker.js]: Unnecessary comment. Please write a comment to `replyIntent` function itself.

**Comment by anonrig** [comment]
Will appreciate if you write a test for this function.

**Comment by RafaelCosman** [comment]
Closing this PR. Please see discussion in https://dev4slack.slack.com/messages/botkit-wit/.

**Pull request closed without merging**